### PR TITLE
Update licenses.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated navigation to get license information

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/3154

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/fundamentals/licenses.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ To confirm your Microsoft Intune license or trial, use the following steps:
 To check on your Azure AD Premium license, use the following steps:
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
-2. Select **Azure Active Directory**. Under **Basic information**, view your license.
+2. Select **Azure Active Directory**. Under **Manage** > **Select Licenses** > **All products** > **Select the License** and choose **Service Plan details** to view your license.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/licenses.md:129]:
```suggestion
2. Select **Azure Active Directory**.
3. Select **Manage** > **Select Licenses** > **All products** > **Select the License** and choose **Service Plan details** to view your license.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and Signoff this PR. Thanks

**Comment by Erikre** [comment]
#sign-off

**Comment by traya1** [comment]
Learn Build status updates of commit _[b32463d](https://github.com/msbemba/memdocs/commits/b32463d07ca25e9ff31601676235fca5ab446900)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[memdocs/intune/fundamentals/licenses.md](https://github.com/msbemba/memdocs/blob/patch-154/memdocs/intune/fundamentals/licenses.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/aa268e4d-aaaf-30c8-4118-35d62ca794c9/202212222011267557-3186/BuildReport?accessString=836108869c06ac343318a55330ab83c243ddae04cc4864a77fdaec52e92a4782).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/aa268e4d-aaaf-30c8-4118-35d62ca794c9?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**